server/job/schedule.ts

```diff
@@ -17,17 +18,22 @@ interface ScheduledJob {
 export const scheduledJobs: ScheduledJob[] = [];
 
 export const startJobs = (): void => {
+  const jobs = getSettings().jobs;
+
   // Run recently added plex scan every 5 minutes
   scheduledJobs.push({
     id: 'plex-recently-added-scan',
     name: 'Plex Recently Added Scan',
     type: 'process',
-    job: schedule.scheduleJob('0 */5 * * * *', () => {
-      logger.info('Starting scheduled job: Plex Recently Added Scan', {
-        label: 'Jobs',
-      });
-      plexRecentScanner.run();
-    }),
+    job: schedule.scheduleJob(
+      jobs['Plex Recently Added Scan']?.schedule ?? '0 */5 * * * *',
```

Comment:
Here is an example where having a record with strict keys makes you not have to check if this is undefined.

We can set the defaults job values in settings as well and avoid using null coalescing here if schedule is not set?